src/Plan/src/TermMatchTreeConverter.cpp

```diff
@@ -86,7 +86,7 @@ namespace BitFunnel
 
     const RowMatchNode* TermMatchTreeConverter::BuildDocumentActiveMatchNode()
     {
-        const Term documentActiveDocumentTerm = m_index.GetTermTable0().GetDocumentActiveTerm();
+        const Term documentActiveDocumentTerm = Term(ITermTable::SystemTerm::DocumentActive, 0, 0);
```

Comment:
I'm not sure whether it's a good idea to allow the recipe for the DocumentActive term to leak out into TermMatchTreeConverter. What would you think of moving the private static TermTable::CreateSystemTerm(SystemTerm term) method up to a public static method of ITermTable? It is a little weird putting a static method on an interface and by doing this, we're constraining the implementation of this method, even though we're giving implementers of ITermTable wide lattitude elsewhere.